samples/XCT.Sample/Pages/Views/Calendar/DateTimeOffsetToDayConverter.cs

```diff
@@ -0,0 +1,22 @@
+using System;
```

Comment:
@DamianAntonowicz What about adding the converters as part of XCT converters? Not just in the Sample. It seems to be useful to XCT users.

Reply:
@GUOLDEV I think these converters are just too simple to be worth adding them to XCT